README.md

```diff
@@ -81,8 +81,11 @@ person = ResolverMap("Person")
 def resolve_person_fullname(person, *_):
     return "%s %s" % (person["firstName"], person["lastName"])
 
-# Create and run dev server that provides api browser
-start_simple_server(type_defs, [query, person]) # Visit http://127.0.0.1:8888 to see API browser!
+# Create executable GraphQL schema
+schema = make_executable_schema(type_defs, [query, person])
+
+# Run dev server that provides api browser
+start_simple_server(schema) # Visit http://127.0.0.1:8888 to see API browser!
```

Comment:
Not sure, but shouldn't we say this instead? Unless "API browser" is actually a name" (same thing for the line above).
```suggestion
start_simple_server(schema)  # Visit http://127.0.0.1:8888 to see the API explorer!
```

Reply:
I guess I was trying to dodge this conversation by not changing that wording in this Pull Request.  ;)

But you're right, the wording could be improved.